integ-test/src/test/java/org/opensearch/sql/sql/GeoPointIT.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+import java.util.Map;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_GEOPOINT;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+
+public class GeoPointIT extends SQLIntegTestCase {
+  @Override
+  protected void init() throws Exception {
+    loadIndex(Index.GEOPOINT);
+  }
+
+  @Test
+  public void test_geo_point() {
+    String query = "SELECT geo_point_object FROM " + TEST_INDEX_GEOPOINT;
+    JSONObject result = executeJdbcRequest(query);
+    verifyDataRows(result,
+        rows(new JSONObject(Map.of(
+                "lat", 40.71,
+                "lon", 74))),
+        rows(new JSONObject(Map.of(
+            "lat", -33.85253637358241,
+            "lon", 151.21652352950258))),
+        rows(JSONObject.NULL)
+    );
+  }
+
+  @Test
+  public void test_geo_point_unsupported_format() {
+    String query = "SELECT geo_point_geohash FROM " + TEST_INDEX_GEOPOINT;
```

Comment:
Can we add examples for all the supported geo_point formats (like the alternate geo_point_string_point) or known locations? 
We could potentially support these later and updated the tests.